_component_design/chips.md

```diff
@@ -13,32 +13,87 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
+## Introduction
+
 <div class="hxRow">
-{% column left:"hxCol" %}
-{% figure [caption:"Basic"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/chips.svg){:width="100%"}
-{% endfigure %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- Chips are best used with indicating a list of items that may be part of a different workflow. For example, requestors on tickets can be added and removed or management tasks that can be performed by requestors. Devices also offer a good option for chip use. 
+
+
+### Best practices
+
+- Avoid using chips in multiply locations on apage. 
+- Make sure the user is well aware of what the chip is. 
```

Comment:
What does this mean?

Reply:
Lets try, "- Ensure the user can concretely identify the object represented by the chip."

Reply:
s/object/item